cluster/gce/gci/configure-kubeapiserver.sh

```diff
@@ -110,15 +110,26 @@ function start-kube-apiserver {
   fi
   if [[ -n "${NUM_NODES:-}" ]]; then
     # If the cluster is large, increase max-requests-inflight limit in apiserver.
-    if [[ "${NUM_NODES}" -gt 3000 ]]; then
-      params+=" --max-requests-inflight=3000 --max-mutating-requests-inflight=1000"
-    elif [[ "${NUM_NODES}" -gt 500 ]]; then
-      params+=" --max-requests-inflight=1500 --max-mutating-requests-inflight=500"
+    if [[ ! "${params}" =~ "--max-requests-inflight[= ]" ]]; then
```

Comment:
Can we do that in slightly more generic way?
What about introducing a function like `appendParamIfNotPresent` and use it here?

[Longer-term we ideally should use it for all params, but that obviously doesn't have to be part of this PR.]

Reply:
Done